.github/workflows/ci.yml

```
@@ -10,6 +10,22 @@ jobs:
   build:
 
     runs-on: macos-11
```

Following comment to rename the job identifier, lets also specify a friendlier name:

```suggestion
    name: Checks (Xcode ${{ matrix.xcode_version }})
    runs-on: macos-11
```